packages/office-ui-fabric-react/src/components/Separator/Separator.base.tsx

```diff
@@ -13,19 +13,19 @@ export class SeparatorBase extends BaseComponent<ISeparatorProps, {}> {
   }
 
   public render(): JSX.Element {
-    const { text, styles, theme, className, vertical, alignText } = this.props;
+    const { styles, theme, className, vertical, alignContent } = this.props;
 
     this._classNames = getClassNames(styles!, {
```

Comment:
`_classNames` is never read anywhere, just written to. I think it seconds the notion of this being a functional component.